amodem/main.py

```diff
@@ -59,7 +59,8 @@ def recv(config, src, dst, dump_audio=None, pylab=None):
         gain = 1.0 / amplitude
         log.debug('Gain correction: %.3f', gain)
 
-        sampler = sampling.Sampler(signal, sampling.Interpolator(), freq=freq)
+        sampler = sampling.Sampler(signal, sampling.defaultInterpolator,
```

Comment:
I'm not sure why this change speeds up `recv()`.

Reply:
Isn't the `sampling.Interpolator()` object is created once and used once (in both cases)?
Maybe I'm missing something...